Documentation/ceph-cluster-crd.md

```diff
@@ -216,7 +218,9 @@ For more details on the mons and when to choose a number other than `3`, see the
   * `config`: Config settings applied to all OSDs on the node unless overridden by `devices`. See the [config settings](#osd-configuration-settings) below.
   * [storage selection settings](#storage-selection-settings)
   * [Storage Class Device Sets](#storage-class-device-sets)
-* `disruptionManagement`: The section for configuring management of daemon disruptions
```

Comment:
@travisn Was this an intended deletion from the docs?

Reply:
Good catch, not intended. @subhamkrai Could you add this back? Thanks!

Reply:
I was quite confused while reading the docs and trying to decipher where the `disruptionManagement` key should be placed 😅

Reply:
added back. Thanks, @hjkatz